src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationService.kt

```diff
@@ -84,7 +85,7 @@ class NotificationService(val shiftNotificationRepository: ShiftNotificationRepo
         log.debug("Finding unprocessedOnly: $unprocessedOnly notifications between $from and $to for $quantumId")
         val notifications = shiftNotificationRepository.findAllByQuantumIdAndShiftModifiedIsBetween(quantumId, from, to).filter { !unprocessedOnly || (unprocessedOnly && !it.processed) }
         log.info("Found ${notifications.size} unprocessedOnly: $unprocessedOnly notifications between $from and $to for $quantumId")
-        val notificationDtos = notifications.map { NotificationDto.from(it, getNotificationDescription(it)) }
+        val notificationDtos = notifications.map { NotificationDto.from(it, getNotificationDescription(ShiftNotificationType.from(it.shiftType), ShiftActionType.from(it.actionType), it.shiftDate, CommunicationPreference.NONE, clock)) }
```

Comment:
this isn't overly readable with the nested method calls. Is there a clearer way to do this in kotlin? If not can we split each param onto a separate line for readability?

Reply:
Good point, I'd refactored the NotificationDescription stuff but not seen that I could clean this up as a result.